js/views/modals/editListing/EditListing.js

```diff
@@ -218,14 +218,18 @@ export default class extends BaseModal {
       const slug = this.model.get('slug');
       if (slug) {
         app.router.navigate(`${app.profile.id}/store/${slug}`, { trigger: true });
+      } else {
+        throw new Error('There is no slug for this listing in order to navigate!');
       }
     }
   }
 
   onClickViewListingOnWeb() {
     const slug = this.model.get('slug');
     if (slug) {
-      openExternal(`http://openbazaar.com/store/${app.profile.id}/${slug}`);
```

Comment:
Yeah, once we revert back the signature of openExternal, this could be reverted back to. IMHO, callers of openExternal shouldn't need to create an anchor element.